src/build_tools/hooks_hydra.py

```diff
@@ -429,19 +430,31 @@ def pre_module_hook(self, *args, **kwargs):  # pylint: disable=unused-argument
     self.cfg.enable_templating = en_templ
 
 
-def end_hook():
-    """Hook to run shortly before EasyBuild completes"""
+def post_build_and_install_loop_hook(ecs_with_res):
+    """Hook to run after all easyconfigs have been built and installed"""
 
     logger = fancylogger.getLogger()
     fancylogger.logToScreen(True, stdout=True)
     fancylogger.setLogLevelInfo()
 
+    do_lmod_cache = True
+
+    if os.getenv('BUILD_TOOLS_RUN_LMOD_CACHE', '1') != '1':
```

Comment:
3 notes:
- i have a bug, it should be
```
if os.getenv('BUILD_TOOLS_RUN_LMOD_CACHE', '0') != '1':
```
- i really want to use `BUILD_TOOLS_RUN_LMOD_CACHE`, because the hooks are also meant to be used locally. it would be annoying if everyone suddenly has to set an envvar to use the hooks.
- it is you who recommended using just `'1'` as the value to check and nothing else :). i also don't really see what's so confusing about it?

Reply:
Maybe I'm missing something here but I'm only getting more confused :p 

By default, unless we tell it otherwise, the Lmod cache should be generated right? And the way to not run the Lmod cache is by setting the env variable `BUILD_TOOLS_RUN_LMOD_CACHE` to 0? If that var is not set, it should also be run?

Reply:
as discussed, i'll change the default value of `do_lmod_cache` to `False` to avoid confusion